perun-registrar-lib/src/main/java/cz/metacentrum/perun/registrar/impl/RegistrarManagerImpl.java

```diff
@@ -1955,7 +1955,7 @@ private void submitEmbeddedGroupApplications(PerunSession sess, List<Group> grou
 				groupApplication.setExtSourceType(app.getExtSourceType());
 				groupApplication.setCreatedBy("Automatically generated");
 
-				submitApplication(sess, groupApplication, new ArrayList<>());
+				submitApplication(registrarSession, groupApplication, new ArrayList<>());
```

Comment:
registrarSession always has LOA 0, which could be different from the LOA of the actual applicant and result in false rejection based on the membership rules of group/vo, no? Might be wrong, got this solely through static code review (`canBeMember` method)

Reply:
Yes, It might have other side effects, that's why I suggested it should be thoroughly tested. We should probably update `submitApplication()` logic to properly handle embedded applications.

Reply:
I tested it and it works correctly also with stronger LOA rules set on a group layer. But we can discuss if some other use cases are worth testing.